package.json

```diff
@@ -29,7 +32,7 @@
     "check:peerDeps": "gulp checkPeerDependencies # validate peer deps for components; escapes before running lerna if they are not valid",
     "ci-all": "yarn build",
     "clean": "gulp clean",
-    "dev": "NODE_ENV=development BROWSERSYNC_OPEN=true gulp dev",
+    "dev": "yarn dev:docs",
```

Comment:
```suggestion
    "dev": "run-p dev:components dev:docs",
```

Reply:
We should do sequential build here since dev:docs will open the port before the build and the site will not have its resources ready.